src/jobs/deploy-node-project.yml

```diff
@@ -0,0 +1,60 @@
+description: |
+  Build, test and deploy a basic NodeJS project.
+  This job runs `npm install`, `npm run build` and `npm run test`,
+  and on success, runs `npm ci --production` and deploys the current project to the requested instance.
+
+executor: default
```

Comment:
In this current design, without a version parameter listed, there is no way for the user to specify which version of NodeJS to use. 

As there is a single job and no requirements in the executor that are specific to this orb or job, I recommend removing the executor and instead replacing this with the regular Docker executor and instead parameterizing the docker image for the job so that the user may change it if desired.

Example: https://github.com/CircleCI-Public/aws-elastic-beanstalk-orb/blob/master/src/jobs/deploy.yml#L31